script.js

```diff
@@ -1,4 +1,52 @@
+var scissors = "scissors";
+var stone = "stone";
+var paper = "paper";
+var totalScore = 0;
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  var numberGenerator = Math.random() * 3;
+  var computerOption = Math.floor(numberGenerator);
+  //If input equals 0  computer chooses scissors
+  if (computerOption == "0") {
+    //input = scissors
+    if (input == scissors) {
+      return (myOutputValue = "You Draw ");
+    } else if (input == stone) {
+      return (myOutputValue = "You Lose ");
+    } else if (input == paper) {
+      totalScore = totalScore + 1;
+      return (myOutputValue = "You Win " + totalScore);
+    }
+  }
+  //If input equals 1  computer chooses paper
+  if (computerOption == "1") {
+    //input = paper
+    if (input == scissors) {
+      return (myOutputValue = "You Lose ");
+    } else if (input == stone) {
+      totalScore = totalScore + 1;
+      return (myOutputValue = "You Win " + totalScore);
+    } else if (input == paper) {
+      return (myOutputValue = "You Draw ");
+    }
+  }
+  //If input equals 2  computer chooses stone
+  if (computerOption == "2") {
+    //input = stone
+    if (input == scissors) {
+      totalScore = totalScore + 1;
+      return (myOutputValue = "You Win " + totalScore);
+    } else if (input == stone) {
+      return (myOutputValue = "You Draw ");
+    } else if (input == paper) {
+      return (myOutputValue = "You Lose ");
+    }
+  }
+
```

Comment:
you are comparing computerOption to every single user input option.
As stated above , after giving computer an object you can compare it better.
 if (
    (input== scissors && computerChoice== paper) ||
    (input== paper && computerChoice== stone) ||
    (input== stone && computerChoice == scissors)
  ) {
  console.log('win')
  }
  same goes for lose
  as for drew cases, you just need one if condition
  if(input ==computerChoice){
//because input will either be scissors , paper or stone and if those words matches computerChoice , it's a drew
console.log('draw')
}